system/jlib/jsocket.hpp

```diff
@@ -296,7 +296,7 @@ class jlib_decl ISocket : extends IInterface
                            unsigned timeout) = 0;
     virtual void   read(void* buf, size32_t size) = 0;
     virtual size32_t write(void const* buf, size32_t size) = 0; // returns amount written normally same as in size (see set_nonblock)
-    virtual size32_t writetms(void const* buf, size32_t size, unsigned timeoutms=WAIT_FOREVER) = 0;
+    virtual size32_t writetms(void const* buf, size32_t minSize, size32_t size, unsigned timeoutms=WAIT_FOREVER) = 0;
```

Comment:
Why add a minSize arg to writetms() ?  Is it to match readtms() ?

Reply:
Yes, writetms was not previously used, but to make consistent added so that writetms can write a minimum of minSize before returning.
NB: not used in practice by secure socket (see comment re. SSL_MODE_ENABLE_PARTIAL_WRITE)